lib/casclient/frameworks/rails/filter.rb

```diff
@@ -288,7 +288,7 @@ def single_sign_out(controller)
                 controller.params['logoutRequest'] &&
                 #This next line checks the logoutRequest value for both its regular and URI.escape'd form. I couldn't get
                 #it to work without URI.escaping it from rubycas server's side, this way it will work either way.
-                [controller.params['logoutRequest'],URI.unescape(controller.params['logoutRequest'])].find{|xml| xml =~
+                [controller.params['logoutRequest'],URI.decode_www_form_component(controller.params['logoutRequest'])].find{|xml| xml =~
```

Comment:
I think decode_www_form_component raises and error when unescape just ignored invalid characters. Do we need to recue the error or it is okay if an error is raised?

Reply:
we shouldn't be sending any invalid characters, because this is coming from UA. So I think this should be fine. I'd prefer it to raise an error if UA sends us any weird stuff.